doc/vim-slime.txt

```diff
@@ -57,6 +57,10 @@ benefits of using Vim (familiar environment, syntax highlighting, persistence
                         line, to screen, tmux, or whimrepl. A carriage return
                         is automatically appended.
 
+                                                *:SlimeSend0*
+:SlimeSend0 {text}      Send a single line of text, specified on the command
+                        line, to screen, tmux, or whimrepl.
```

Comment:
Would it be better to add something like "this won't send a carriage return" 🤔

Reply:
Done in 708c244